BorrowBooks/features/authentication/registration_server.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+require_once __DIR__ . '/../../repositories/UserRepository.php';
+require_once __DIR__ . '/../../models/User.php';
+
+use models\User;
+use repositories\UsersRepository;
+
+$username = trim($_POST['username']);
+$email = trim($_POST['email']);
+$password = $_POST['password'];
+$confirm_password = $_POST['confirmpassword'];
+
+function alert($message) {
+    echo "<script type='text/javascript'>alert('$message'); window.location.href = 'registration.php';</script>";
+    exit();
+}
+
+// Validations
+if (empty($email) || empty($password) || empty($username) || empty($confirm_password)) {
+    alert("Моля попълнете всички полета.");
```

Comment:
Registration and login form are in english, however the alerts are in bulgarian. we would probably want to unify that.

Reply:
Currently, the alerts are only in Bulgarian because there are no specific requirements regarding whether the site should be in English or Bulgarian. Other projects have been handled in the same manner, as I have observed in older projects as well. Once we receive clear requirements, we will change the language if needed.